cmd/security-agent/subcommands/start/command.go

```diff
@@ -385,7 +385,7 @@ func setupInternalProfiling(config config.Component) error {
 		v, _ := version.Agent()
 
 		cfgSite := config.GetString(secAgentKey("internal_profiling.site"))
-		cfgURL := config.GetString(secAgentKey("security_agent.internal_profiling.profile_dd_url"))
+		cfgURL := config.GetString(secAgentKey("internal_profiling.profile_dd_url"))
```

Comment:
Let me know if the `security_agent` stutter due to `secAgentKey` was on purpose

Reply:
Definitely not on purpose, thanks !